certbot/certbot/ocsp.py

```diff
@@ -68,8 +68,19 @@ def ocsp_revoked(self, cert):
         :rtype: bool
 
         """
-        cert_path, chain_path = cert.cert_path, cert.chain_path
+        return self.ocsp_revoked_by_paths(cert.cert_path, cert.chain_path)
 
+    def ocsp_revoked_by_paths(self, cert_path, chain_path):
+        # type: (str, str) -> bool
+        """Performs the OCSP revocation check
+
+        :param str cert_path: Certificate filepath
+        :param str chain_path: Certificate chain filepath
+
+        :returns: True if revoked; False if valid or the check failed or cert is expired.
+        :rtype: bool
+
+        """
         if self.broken:
```

Comment:
The other PR moves this after the if-cert-is-expired check; is that necessary to do here as well?

Reply:
No. The two checks are independent.

I personally reordered them to keep the logic the same as what is currently in `ocsp_revoked`, but I can switch it if you prefer that for some reason.

Reply:
works for me.